src/main/resources/lang/de_de.json

```diff
@@ -0,0 +1,121 @@
+{
+	"language": "Deutsch",
+
+	"mapping_format.enigma_file": "Enigma‐Datei",
+	"mapping_format.enigma_directory": "Enigma‐Verzeichnis",
+	"mapping_format.enigma_zip": "Enigma‐ZIP‐Archiv",
+	"mapping_format.tiny_v2": "Tiny‐v2‐Datei",
+	"mapping_format.tiny_file": "Tiny‐Datei",
+	"mapping_format.srg_file": "SRG‐Datei",
+	"mapping_format.proguard": "ProGuard‐Datei",
+	"type.methods": "Methoden",
+	"type.fields": "Felder",
+	"type.parameters": "Parameter",
+	"type.classes": "Klassen",
+
+	"menu.file": "Datei",
+	"menu.file.jar.open": "Java‐Archiv öffnen …",
+	"menu.file.jar.close": "Java‐Archiv schließen",
+	"menu.file.mappings.open": "Zuordnungen laden …",
+	"menu.file.mappings.save": "Zuordnungen speichern",
+	"menu.file.mappings.save_as": "Zuordnungen speichern unter …",
+	"menu.file.mappings.close": "Zuordnungen schließen",
+	"menu.file.mappings.drop": "Ungültige Zuordnungen löschen",
+	"menu.file.export.source": "Quelldateien exportieren …",
+	"menu.file.export.jar": "Java‐Archiv exportieren …",
+	"menu.file.stats": "Statistiken der Zuordnungen …",
+	"menu.file.stats.title": "Enthaltene Elemente auswählen",
+	"menu.file.stats.generate": "Statistiken generieren",
+	"menu.file.exit": "Beenden",
+	"menu.decompiler": "Dekompilierer",
+	"menu.view": "Ansicht",
+	"menu.view.themes": "Farbgestaltung",
+	"menu.view.themes.default": "Standard",
+	"menu.view.themes.darcula": "Darcula",
+	"menu.view.themes.system": "Betriebssystem",
+	"menu.view.themes.none": "Keine (JVM‐Standard)",
+	"menu.view.languages": "Sprache",
+	"menu.view.languages.title": "Sprache ändern",
+	"menu.view.languages.summary": "Die ausgewählte Sprache wird nach dem nächsten Neustart verwendet.",
+	"menu.view.languages.ok": "Okay",
```

Comment:
"Ok" is equally common, especially for dialogs, and shorter
(See also `menu.help.about.ok`, where it's translated with "Ok" already)

Reply:
In fact, I had already standardized this as `„OK“` and don't know how it could be lost again.
Are you sure that `„Ok“` is more common in German than `„OK“`?

Reply:
> Are you sure that `„Ok“` is more common in German than `„OK“`

No. I exemplarily forced such a dialog in Microsoft Word and it did use a capital K.
As long as you don't change it to `O.K.` everything is fine 😛 
![image](https://user-images.githubusercontent.com/12124394/79695024-97092e80-8274-11ea-8ef2-904b4b2f0716.png)

Reply:
Thank you! The translations of the corresponding strings have now been rectified.